oci/layout/oci_transport_test.go

```diff
@@ -151,8 +156,9 @@ func TestReferenceStringWithinTransport(t *testing.T) {
 	defer os.RemoveAll(tmpDir)
 
 	for _, c := range []struct{ input, result string }{
-		{"/dir1:notlatest", "/dir1:notlatest"}, // Explicit tag
-		{"/dir2", "/dir2:latest"},              // Default tag
+		{"/dir1:notlatest:notlatest", "/dir1:notlatest:notlatest"}, // Explicit image
+		{"/dir2:image", "/dir2:image"},                             // Default image
```

Comment:
There is actually nothing “default” about this `:image` now that `:latest` is not treated specially; this line can be deleted.

Reply:
removed default line